lib/elements/fill_stitch.py

```diff
@@ -374,7 +374,7 @@ def staggers(self):
         tooltip=_('The last stitch in each row is quite close to the first stitch in the next row.  '
                   'Skipping it decreases stitch count and density.'),
         type='boolean',
-        sort_index=26,
+        sort_index=30,
```

Comment:
I do hate the sort_index thing so much, even though I'm the one that added it.  I can't wait until we redo the GUI entirely.